schema/schema.sql

```diff
@@ -0,0 +1,48 @@
+CREATE TABLE item (
+    id SERIAL PRIMARY KEY,
+    genre_id INT,
+    author_id INT,
+    label_id INT,
+    publish_date DATE,
+    archived BOOLEAN,
+    FOREIGN KEY (genre_id) REFERENCES genre(id),
+    FOREIGN KEY (author_id) REFERENCES author(id),
+    FOREIGN KEY (label_id) REFERENCES label(id)
+);
+
```

Comment:
Please consider adding the id as a primary key and if you use "id INT GENERATED ALWAYS AS IDENTITY"  I think is a good practice if you keep consistency in your code so it will be good if you use it here too.